EveripediaNetwork/ep-ui
# Fix glossary page stacking order

# Glossary Page 

_This PR fixes the dropdown menu stacked under  the glossary page header section_

## How should this be tested?

1. Navigate to the glossary page
2. Click on any of the dropdown menu[categories or more menu] to confirm the new changes
3. Confirm the changes on the mobile menu with the nav menu opened as well


## Screenshots 


<details>
<summary> Before </summary>

<img width="440" alt="Screenshot 2024-08-28 at 10 15 21" src="https://github.com/user-attachments/assets/bdd19eb0-01b6-4a6f-a78e-eb156a5f3d54">

<img width="1509" alt="Screenshot 2024-08-28 at 10 16 24" src="https://github.com/user-attachments/assets/a0d8b0c9-160d-47c8-bf1e-339305d47edd">

<img width="405" alt="Screenshot 2024-08-28 at 10 21 45" src="https://github.com/user-attachments/assets/e8c7ea01-c1b3-4c2f-9446-4fded9733948">

</details>


<details>
<summary> After </summary>

<img width="1512" alt="Screenshot 2024-08-28 at 10 20 33" src="https://github.com/user-attachments/assets/68db07bd-1144-45a6-97cf-f596d211ff22">
</details>

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3084
